tests/integration/mesos/_test_mesos.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
```

Comment:
Once you got pre-commit set up please run it on all files. These imports are not necessary for Python 3.